### PR TITLE
Add contact-section

### PR DESCRIPTION
**Pull request by zeff96**
Add an index.html file for contact-section to the document.
Add style for contact-section for display, position, fonts, and background color.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -300,13 +300,11 @@ <h2>
...
       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I&#8217;d love to help with it 
     </p>
-    <form action="" class="form">
-      <input type="text" name="name" id="name" class="input-form">
-      <input type="email" name="email" id="email" class="input-form">
-      <textarea name="textarea" id="textarea" class="textarea" placeholder="write your message here"></textarea>
-      <a class="action" href="#">
-        <button class="button">get it touch</button>
-      </a>
+    <form action="https://formspree.io/f/%7Bform_id%7D" method="post" class="form">
```
Comment (brainconnect93) [inline index.html]: - [x] Hi @zeff96, well done working on the contact section of your project, but according to the project requirement, we were asked to create a form-spree in such a way that any visitor can easily contact us when a message is submitted using the form-spree. but currently, the form-spree link is not working, kindly create an account on form-spree and use the id-link given to you in the `form action`

What the form is given when a message is submitted 👇 

hashid

File: index.html [hunk 2]
```diff
@@ -300,13 +300,11 @@ <h2>
...
+      <input type="text" name="name" id="name" class="input-form" placeholder="Enter name" maxlength="30" required autofocus>
+      <input type="email" name="email" id="email" class="input-form" placeholder="Enter email" required autofocus>
+      <textarea name="textarea" id="textarea" class="textarea" placeholder="write your message here" maxlength="500" required autofocus></textarea>
+      <input type="submit" value="submit" class="submit"></input>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest we stick to the Figma design as much as possible. Currently, your form button is having a placeholder name `Submit`. Kindly change it to `Get In Touch` as we have in the Figma design with this you have shown a creative ability to replicate any given to you design by a client. 👍 👍

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @zeff96, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @zeff96 , 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**